code/modules/power/singularity/particle_accelerator/particle_control.dm

```diff
@@ -117,8 +117,9 @@
 		strength++
 		strength_change()
 
-		message_admins("PA Control Computer increased to [strength] by [key_name_admin(usr)](<A HREF='?_src_=holder;adminmoreinfo=\ref[usr]'>?</A>) (<A HREF='?_src_=holder;adminplayerobservefollow=\ref[usr]'>FLW</A>) in ([x],[y],[z] - <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[x];Y=[y];Z=[z]'>JMP</a>)",0,1)
-		log_game("PA Control Computer increased to [strength] by [key_name(usr)] in ([x],[y],[z])")
+		message_admins("PA Control Computer increased to [strength] by \

```

Comment:
They're on linux using vi or something and their screen isn't wide enough, so they break line to fit in their screen.